connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -100,6 +101,33 @@ class DatasetSuite extends ConnectFunSuite with BeforeAndAfterEach {
     assert(actualPlan.equals(expectedPlan))
   }
 
+  test("write jdbc") {
```

Comment:
This seems to break `branch-3.4` somehow. I'm checking it now.
- https://github.com/apache/spark/actions/runs/4414215325/jobs/7742911619
```
[info] - write jdbc *** FAILED *** (527 milliseconds)
[info]   io.grpc.StatusRuntimeException: INTERNAL: No suitable driver
[info]   at io.grpc.Status.asRuntimeException(Status.java:535)
```